indigo/analysis/refs/base.py

```diff
@@ -196,13 +197,21 @@ class SectionRefsFinderENG(BaseInternalRefsFinder):
         r'''\b
         (
           (?P<ref>
-            sections?\s+
+            (?<!-)sections?\s+
             (?P<num>\d+[A-Z0-9]*)  # first section number, including subsections
           )
-          (\s*\([A-Z0-9]+\))*      # bracketed subsections of first number
+          (
+            (\s*(,|(,\s*)?and|(,\s*)?or)\s*)*          # list separators
```

Comment:
How about:

`(\s*(,|and|or))*`

The final `*` allows multiple repetitions, which handles `, and` and `, or` so they don't need to be special cased.

Reply:
ditto for the separators below, too.

Reply:
thanks, yours is much better!